dist/stat_data.py

```diff
@@ -213,6 +213,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_fail_in_reconciliation', 'pages selected for eviction unable to be evicted because of failure in reconciliation'),
     CacheStat('cache_eviction_fail_parent_has_overflow_items', 'pages selected for eviction unable to be evicted as the parent page has overflow items'),
     CacheStat('cache_eviction_force', 'forced eviction - pages selected count'),
+    CacheStat('cache_eviction_force_update_list', 'forced eviction when more than 1000 updates found in an update chain.'),
```

Comment:
Not fond of the statistics naming, maybe we can call it:

`cache_eviction_force_long_update_list`

Also, we should probably not mention the exact number of updates we force the page for eviction. If we tune that value tomorrow, we will have to change the description of the statistics, which is hard-coupled to t2/server-status output. So I suggest something like:

`forced eviction - pages selected because of a large number of updates to a single item`

Reply:
Fixed